tools/download_paks.py

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import argparse
+import fnmatch
+import os
+import posixpath
+import spear
+from tqdm import tqdm
+import urllib.request
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--output_dir", required=True)
+    parser.add_argument("--platform", required=True)
+    parser.add_argument("--version_tag", required=True)
+    parser.add_argument("--scene_ids", required=True)
```

Comment:
- `scene_ids` should be optional, if it is not given then the script should download everything
- it is ok if all possible scene names are hard-coded in this script somewhere